src/hooks/useReceiptData.ts

```diff
@@ -1,20 +1,21 @@
 import { useEffect, useState } from 'react';
-// import utils from 'utils';
+import utils from 'utils';
 
 export const useReceiptData = (transactionId: string) => {
```

Comment:
I don't think this his the best approach. We are calling (by the way multiple times) the API utils.apiClient.transactions.getTransactionReceipt before the user click on the download button trough this hook

Reply:
This is needed so that the button can be disabled accordingly to the availability of the receipt. 
I tested and the API call is made once.

Reply:
I think this is the correct approach. Of course then, after clicking, an error could occurs and the unhappy path should be managed (a console.warn is enough for now)